.github/workflows/push_dockerhub.yml

```
@@ -0,0 +1,39 @@
+name: nf-core Docker push
+# This builds the docker image and pushes it to DockerHub
+# Runs on nf-core repo releases and push event to 'dev' branch (PR merges)
+on:
+  push:
+    branches:
+      - dev
+  release:
+    types: [published]
+
+push_dockerhub:
```

```suggestion
jobs:
  push_dockerhub:
```

And should add an extra 2 spaces indentation to everything below